### PR TITLE
Add notes about offline usage

### PR DESCRIPTION
**Pull request by loganakamatsu**

### CODE REVIEW

File: source/includes/getting-started.md [hunk 1]
```diff
@@ -693,5 +693,20 @@ In order to begin the second Route, you'll need to finish the first one by makin
 
 When all the routes have been finished, your [RouteStateListener](https://foxtrotsystems.github.io/android-sdk-javadoc/io/foxtrot/android/sdk/state/RouteStateListener.html) will receive a call to `onAllRoutesFinished()` and you'll have the opportunity to respond.
 
-### Understanding Routes
+
+## Offline Functionality
+After initial setup and login, the Foxtrot SDK can function without a constant internet connection. In some cases while the SDK does not have an internet connection, Foxtrot will be unable to change the route or update the ETA's, however, data collection will continue to function as normal Once connectivity has been re-established, the cached data will be sent to Foxtrot Servers and the SDK will receive continuous route optimization.
```
Comment (hmvien) [inline source/includes/getting-started.md]: Missing period after `normal` and before `Once`.

File: source/includes/getting-started.md [hunk 2]
```diff
@@ -693,5 +693,20 @@ In order to begin the second Route, you'll need to finish the first one by makin
...
+
+### When an Internet Connection is Required:
+The first driver login requires a working internet connection. A connection is necessary to verify the validity of the driver and their configuration. 
+After the first login, drivers can be logged in again without an internet connection as long as no calls to logout() are made in between. One example where this might happen is if your app has been restored from the background and you're not sure whether or not the app was killed (for example due to low memory in the operating system). Calling login() again on app resume does not require an active connection.
```
Comment (hmvien) [inline source/includes/getting-started.md]: `After the first login, drivers can be logged in again without an internet connection as long as the same `Driver` is used and no calls to logout() has been made.`

Comment (hmvien) [reply source/includes/getting-started.md]: I think the `Calling login() again on app resume does not require an active connection.` is redundant

**Comment by loganakamatsu** [comment]
Thanks @hmvien -ster, added 5444a39

**Comment by hmvien** [comment]
:shipit:

**Pull request merged**